src/com/company/assignments/Homework.java

```diff
@@ -0,0 +1,20 @@
+package com.company.assignments;
+
+import java.util.Date;
+
+public class Homework extends Assignment{

```

Comment:
[Fix] ce aduce aceasta clasa in plus fata de Assignement? daca e doar de nume, se putea adauga un camp "type" in assignment. Daca nu contine campuri extra, care sa justifice existenta, atunci nu ii vad rostul

Reply:
Clasa Assignment trebuia sa fie o clasa abstracta, de aceea am adaugat si clasa Homework chiar daca nu are atribute in plus, dar nu cred ca am facut-o abstracta. Ce pot sa fac pentru a rezolva?